app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for human in people:
+        person_list.append(Person(name=human["name"], age=human["age"]))
+
+    for person in people:
+        if "wife" in person and person["wife"]:
```

Comment:
Use walrus operator and simpify code:
```
if wife := person.get("wife"):
    current_person.wife = Person.people[wife]
```

Do the same with husband

Reply:
@pavlopro don`t understand

Reply:
why?

Reply:
@pavlopro Error

Reply:
![image](https://github.com/user-attachments/assets/dd914894-cd5e-4340-9f1e-306862b08672)